mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (result) {
+      return res.status(404).json({ message: "Order not located" });
+    }
+    res.status(200).json({ message: "Order eradicated successfully" });
+  } catch (err) {
+    res.status(500).json({ message: errorHandler(err) });
```

Comment:
In the error handling, it would be better to include the actual error message along with the error handler's output. This will provide more information about the error to the client. 

Change it to: 
res.status(500).json({ message: errorHandler(err), error: err.message });